src/control/drpc/property.go

```diff
@@ -39,6 +41,12 @@ const (
 	PoolPropertyOwnerGroup = C.DAOS_PROP_PO_OWNER_GROUP
 	// PoolPropertyECCellSize is the EC Cell size.
 	PoolPropertyECCellSize = C.DAOS_PROP_PO_EC_CELL_SZ
+	// PoolPropertyRedunFac
+	PoolPropertyRedunFac = C.DAOS_PROP_PO_REDUN_FAC
+	//PoolPropertyECPda is performance domain affinity level of EC object.
+	PoolPropertyECPda = C.DAOS_PROP_PO_EC_PDA
+	//PoolPropertyRPPda is performance domain affinity level of replicated object.
```

Comment:
Not sure if gofmt will yell at you, but a space is usually expected after the "//" for comments.